### ISSUE TITLE
@atomicRmw .Min and .Max unavailable on mipsel

### ISSUE THREAD

**Issue by Vexu** [ziglang/zig#4724]
```
LLVM ERROR: Cannot select: 0x55b11a042dd0: i32,ch = AtomicLoadUMin<(load store monotonic 1 on %ir.x)> 0x55b11a170690, FrameIndex:i32<0>, Constant:i32<1>, atomics.zig:195:9
  0x55b11a21fec0: i32 = FrameIndex<0>
  0x55b11a17f2c8: i32 = Constant<1>
```
```
LLVM ERROR: Cannot select: 0x55cab0d24320: i32,ch = AtomicLoadUMax<(load store monotonic 1 on %ir.x)> 0x55cab0eb6e38, FrameIndex:i32<0>, Constant:i32<1>, atomics.zig:193:9
  0x55cab10f6fe0: i32 = FrameIndex<0>
  0x55cab11baa98: i32 = Constant<1>
```

Related #3345

**Comment by Vexu** [comment]
Added in llvm 10 https://github.com/llvm/llvm-project/commit/d7357c52a40a136f25c1cf5ae31a699d51885e49

**Comment by andrewrk** [comment]
Fixed with the merge of llvm10 branch: 93c7fa105f9cc4dbb36192469ecff47102d955d6
